components/SideNavbar.js

```diff
@@ -3,38 +3,35 @@ import React from 'react';
 import { Routes } from '../models/routes';
 
 function SideNavbar({ show, setShow }) {
+  const linkClasses = 'btn btn-ghost btn-sm rounded-btn shadow-none';
```

Comment:
Hi @fyvfyv. So we have another issue pertaining to this concerning the use of directives in Tailwind. See #225